packages/plugin-toolbar/assets/Toolbar.xml

```diff
@@ -37,7 +37,7 @@
                             </select>
                         </t>
                         <t t-else="">
-                            <button t-attf-class="fas {{toolbarItem.class}} fa-fw"
+                            <button t-attf-class="fa {{toolbarItem.class}} fa-fw"
```

Comment:
I guess Odoo does not use the same version of FontAwesome as the one @Zinston  used ?

Reply:
This is the previous version of FontAwesome indeed. I thought Odoo had upgraded to FA5 though...